sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -146,5 +147,5 @@ void alterNamespace(
    * @throws NoSuchNamespaceException If the namespace does not exist (optional)
    * @throws UnsupportedOperationException If drop is not a supported operation
```

Comment:
let's add parameter doc for `cascade`

Reply:
let's mention that implementation should throw `NonEmptyNamespace` exception if the namespace is not empty and `cascade = false`.